src/features/app/home/Home.js

```diff
@@ -0,0 +1,31 @@
+import { IconCheck } from './_components/IconCheck'
+import { IconLock } from './_components/IconLock'
+import { TaxFormSection } from './_components/TaxFormSection'
+import { PostponeSection } from './_components/PostponeSection'
+import styles from './Home.module.css'
+
+
```

Comment:
pouzivame .tsx v celom projekte

Reply:
nemam uplne skusenost s js vs tsx, ked to islo tak som to neriesil, ale vies mi poradit co vsetko premenovat? a pride ti to kriticke brzdit tym merge ak teda nam ide o cas? lebo to sa da otvorit PR hned ako dalsi step

Reply:
https://www.w3schools.com/typescript/typescript_intro.php

vsetky subory by mali mat koncovku ```.tsx``` ak pouzivaju jsx alebo ```.ts``` ak v nich nieje jsx